content/docs/custom-plugins/configuring/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Configuring Custom Plugins
+lastUpdated: '2021-06-17T12:11:00.0Z'
+description: How to configure a custom Backstage plugin to Roadie
+---
+
+## Introduction
+
+Roadie supports all standard Backstage plugin types that can be built using instructions defined in [Backstage documentation][backstage-plugin-documentation]. Your plugins can be published via npm or yarn, like publishing a normal package. We will provide you functionality to automatically add custom plugins from repositories to your Roadie instance. If you want to have a private hosted repository for your Roadie plugins with automatic update workflow you can follow instructions on [setting up private secure Artifactory for Roadie plugins.](/docs/custom-plugins/artifactory/)
```

Comment:
> Roadie supports all standard Backstage plugin types

We don't support backend plugins yet.

> If you want

I would split out a new paragraph from this point on. The topic changes slightly at this point.